src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java

```diff
@@ -50,4 +52,15 @@ public Movie add(Movie movie) {
             throw new DataProcessingException("Can't get movies", e);
         }
     }
+
+    @Override
+    public Optional<Movie> getById(Long movieId) {
+        try (Session session = sessionFactory.openSession()) {
+            Query query = session.createQuery("FROM Movie WHERE movieId = :movieId");
```

Comment:
```suggestion
            Query<Movie> query = session.createQuery("FROM Movie WHERE movieId = :movieId", Movie.class);
```
using this you can avoid casting a line below

Reply:
Okay, thanks a lot